### DISCUSSION TITLE
How does it work?

### DISCUSSION THREAD

**Discussion by Bobbaone** [eizedev/AirConnect-Synology#45] [Q&A] [answered]
Hello,

ive installed it, airconnect is running.

But how is it possible to Stream to my Sonos Boxes via Windows PC?

There is no Interface - i cant read it on the readme - is there a File where i can find the way to streaming to my Sonos Boxes?

Greetings

**Comment by eizedev** [comment, marked as answer]
Hi @Bobbaone 

the client from where you want to stream music to sonos needs to be compatible with the airplay protocol if you want to use the AirConnect "server".  
This is by default only available on all apple devices. On Windows you could of course use the [Sonos windows app](https://support.sonos.com/s/downloads?language=en_US). This is the same sonos app as on your mobile device, so you do not need to use any other application or the airplay protocol.

Also a few other applications will support airplay. On windows the easiest way is to use iTunes. iTunes supports airplay by default.
